e2e/utils/CommonActions.js

```diff
@@ -58,8 +67,8 @@ class CommonActions {
      * @param elementCSS WebElement locator.
      */
     static setInputTextField(elementCSS, value) {
-        this.waitElement(elementCSS);
-        this.moveToComponent(elementCSS);
+        browser.waitForExist(elementCSS);
+        //browser.moveToComponent(elementCSS);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3ACommentedCode)